controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -156,8 +155,8 @@ def test_create_snapshot_no_source_volume(self):
     @patch("controller.controller_server.csi_controller_server.get_agent")
     def test_create_snapshot_source_volume_not_found(self, storage_agent):
         storage_agent.return_value = self.storage_agent
-        self.mediator.get_volume_name = Mock()
-        self.mediator.get_volume_name.side_effect = array_errors.ObjectNotFoundError("volume_id")
+        self.mediator.get_object_by_id = Mock()
+        self.mediator.get_object_by_id.side_effect = array_errors.ObjectNotFoundError("volume_id")

```

Comment:
this is not allowed by the mediator interface

Reply:
done